src/overlays/actors/ovl_Obj_Switch/z_obj_switch.c

```diff
@@ -10,10 +10,11 @@
 
 #define FLAGS ACTOR_FLAG_4
 
-#define OBJSWITCH_TYPE(thisx) ((thisx)->params & 7)
-#define OBJSWITCH_SUBTYPE(thisx) (((thisx)->params >> 4) & 7)
-#define OBJSWITCH_SWITCH_FLAG(thisx) (((thisx)->params >> 8) & 0x3F)
-#define OBJSWITCH_FROZEN(thisx) (((thisx)->params >> 7) & 1)
+#define OBJSWITCH_TYPE(thisx) PARAMS_GET((thisx)->params, 0, 3)
+#define OBJSWITCH_SUBTYPE(thisx) PARAMS_GET((thisx)->params, 4, 3)
+#define OBJSWITCH_SWITCH_FLAG(thisx) PARAMS_GET((thisx)->params, 8, 6)
+#define OBJSWITCH_FROZEN(thisx) PARAMS_GET((thisx)->params, 7, 1)
+
```

Comment:
if we can, for any already existing actor-specific macros, would like them to say `_GET_` in them so we can standardize that now

Reply:
what about booleans like "frozen" here, should we still use `_GET_` or `_IS_` ?
another example is `ENDOOR_IS_DOUBLE_DOOR`, should it be `ENDOOR_GET_IS_DOUBLE_DOOR` or?

Reply:
id say it depends on how its named. if you imagine the "set" macro that actually sets the params in the actor list:
`OBJSWITCH_PARAMS(type, subtype, switchFlag, isFrozen)`
I think it should be `_GET_<NAME>`, whatever the variable name is